front/components/providers/AI21Setup.jsx

```diff
@@ -50,19 +50,21 @@ export default function AI21Setup({ owner, open, setOpen, config, enabled }) {
         }),
       }),
     });
-    let data = await res.json();
+    await res.json();
```

Comment:
Should we move those setState (eg `setEnableRunning(false))` before the await since they were not waiting for it before nor conditioned by the I/O result?

Reply:
I think it's fine to keep it here. We'll probably want to handle errors at some point but fine to leave it as is for now :+1: